lib/maze/browser_stack_utils.rb

```diff
@@ -59,6 +60,12 @@ def upload_app(username, access_key, app)
           $logger.info "app uploaded to: #{app_url}"
           $logger.info 'You can use this url to avoid uploading the same app more than once.'
         end
+
+        unless app_id_file.nil?
+          $logger.info "writing uploaded app url to #{app_id_file}"
+          File.write(app_id_file, app_url)
```

Comment:
```suggestion
          File.write(Maze::Helper.expand_path(app_id_file), app_url)
```

Then you can use `~` in the id file path.